storage/tests/run_integration_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+delay=1
+connected=no
+readonly ATTEMPTS=$(seq 1 8)
+for attempt in $ATTEMPTS; do
+  if curl "https://nghttp2.org/httpbin/get" >/dev/null 2>&1; then
```

Comment:
Any reason for picking this particular URL?

Reply:
Yes, you can get the contents of your request in the response, and simulate failures, and return blobs of data too:

https://nghttp2.org/httpbin/

Makes it convenient for small tests like this, but we should be nice and not abuse their site.